src/legacy/core_plugins/vis_default_editor/public/_sidebar.scss

```diff
@@ -59,15 +59,6 @@
   + .visEditorSidebar__section {
     margin-top: $euiSizeS;
   }
-
-  label:not([class^='eui']) {
-    @include __legacyLabelStyles__bad;
```

Comment:
This is removed due to no necessity anymore. All the label were removed during `EUIfication`